packages/web-frontend/cypress/support/helpers.js

```diff
@@ -21,6 +23,13 @@ export class EmptyConfigError extends Error {
   }
 }
 
+export class TestUserPasswordUndefinedError extends Error {
+  constructor() {
+    super('Please specify a value for CYPRESS_TEST_USER_PASSWORD in packages/web-frontend/.env');
+    this.name = 'TestUserPasswordUndefinedError';
```

Comment:
For my own interest - does setting a name on an error give nicer console logs?

Reply:
Yeap, the stack trace of an uncaught error will be like this:

![image](https://user-images.githubusercontent.com/20692464/98485172-8cda6c00-2268-11eb-80ab-7c9eefbfaf29.png)

instead of just `Error: ...`

Reply:
Very cool, and good to know